portfolio/src/main/webapp/about.js

```diff
@@ -1,82 +1,106 @@
+ */
+function next()  {
+    let gameStatus = document.getElementById('game-status');
+    if (gameStatus.innerHTML=='Press Start to Begin!') {
+        gameStatus.innerHTML = 1;
+        document.getElementById('advance-game-btn').innerHTML = 'Next';
+    } else {
+        let chosenAnswer = -1;
+        const chosenRadios = document.querySelectorAll('input[name="selected-answer"]');
+        for (chosenRadio of chosenRadios) {
+            if(chosenRadio.checked) {
+                chosenAnswer = chosenRadio.value;
+            }
+        }
+        if (chosenAnswer != -1) {
+            if (correctAnswer == chosenAnswer) {
+                alert('This is the lie!');
+            } else {
+                alert('This is the truth!');
+            }
+        }
+    } 
+    const truthIndices = loadInTruth();
+    correctAnswer = loadInLie(truthIndices);
 }
 
+/**
+ * Populates the page with two true facts in randomized indexes
+ * 
+ *@return {Array} an array populated with two numbers containing
+ * locations of the true facts
+ */
 function loadInTruth() {
 
-    let first_truth_value = Math.floor(Math.random() * truths.length);
-    let second_truth_value = Math.floor(Math.random() * truths.length);
-    while (first_truth_value == second_truth_value) {
-        second_truth_value = Math.floor(Math.random() * truths.length);
+    let firstTruthValue = Math.floor(Math.random() * truths.length);
+    let secondTruthValue = Math.floor(Math.random() * truths.length);
+    while (firstTruthValue == secondTruthValue) {
+        secondTruthValue = Math.floor(Math.random() * truths.length);
     }
 
-    let first_truth_index = Math.floor(Math.random() * 3);
-    let second_truth_index = Math.floor(Math.random() * 3);
-    while (first_truth_index == second_truth_index) {
-        second_truth_index = Math.floor(Math.random() * 3);
+    let firstTruthElementIndex = Math.floor(Math.random() * 3);
+    let secondTruthElementIndex = Math.floor(Math.random() * 3);
+    while (firstTruthElementIndex == secondTruthElementIndex) {
+        secondTruthElementIndex = Math.floor(Math.random() * 3);
     }
 
-    loadInFact(first_truth_index, first_truth_value, true);
-    loadInFact(second_truth_index, second_truth_value, true);
+    loadInFact(firstTruthElementIndex, firstTruthValue, true);
+    loadInFact(secondTruthElementIndex, secondTruthValue, true);
 
-    return [first_truth_index, second_truth_index];
+    return [firstTruthElementIndex, secondTruthElementIndex];
 }
 
 
-function loadInLie(truth_indeces) {
+/**
+ * Gets the image and text from the position stored in index and 
+ * populates it with the fact stored in factIndex.  Populates a truth
+ * if isTruth is true and a lie otherwise
+ * 
+ *@param {number} index the position we will populate the fact
+ *@param {number} factIndex the position in the array of the 
+ * fact that will be populated
+ *@param {boolean} isTruth true if we will populate a truth; 
```

Comment:
More nitpicking... add a space between the asterisk and the at signs.

Reply:
Added space